test/e2e/argo-workflows/dca-hpa.yaml

```diff
@@ -0,0 +1,905 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: argo-datadog-cluster-agent-hpa
+spec:
+  entrypoint: main
+  #onExit: delete # call argo submit --entrypoint delete instead
+  arguments:
+    parameters:
+    - name: cluster-agent-rbac-cluster-role
+      value: |
+        apiVersion: rbac.authorization.k8s.io/v1
+        kind: ClusterRole
+        metadata:
+          name: dca
+        rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - events
+          - endpoints
+          - pods
+          - nodes
+          - componentstatuses
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - "autoscaling"
+          resources:
+          - horizontalpodautoscalers
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          resourceNames:
+          - datadogtoken
+          verbs:
+          - get
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - update
+          - create
+
+    - name: cluster-agent-rbac-sa
+      value: |
+        kind: ServiceAccount
+        apiVersion: v1
+        metadata:
+          name: dca
+          namespace: default
+
+    - name: cluster-agent-rbac-cluster-role-binding
+      value: |
+          apiVersion: rbac.authorization.k8s.io/v1
+          kind: ClusterRoleBinding
+          metadata:
+            name: dca
+          roleRef:
+            apiGroup: rbac.authorization.k8s.io
+            kind: ClusterRole
+            name: dca
+          subjects:
+          - kind: ServiceAccount
+            name: dca
+            namespace: default
+
+    - name: hpa-rbac
```

Comment:
maybe `custom-metrics-rbac`?

Reply:
I think we need to be more specific than custom metrics. I'm updating to `external-custom-metrics-rbac`